cmd/secret-manager/commands/login.py

```
@@ -0,0 +1,42 @@
+
+@click.command()
+def login():
+    """Authenticate to Google Cloud."""
+
+    click.echo("Authenticating to Google Cloud...\n")
+
+    try:
+        # Run gcloud interactively with error-level verbosity to suppress warnings
+        # (we set the quota project immediately after, so the warning is not relevant)
+        subprocess.run(
+            ["gcloud", "auth", "application-default", "login", "--verbosity=error"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)